.gitignore

```diff
@@ -5,6 +5,7 @@
 .coverage
 .pytest_cache
 .tox
+.cache
```

Comment:
This is from an older version of pytest. The newer version is `.pytest_cache` and is already in this file. Pleas remove this when you can, thanks!